clock.py

```diff
@@ -185,8 +185,8 @@ def __init__(self, handle):
 
         # Some hardware cannot keep two GStreamer playback pipelines active
         try:
-            model = file('/proc/device-tree/openprom/model', 'r').readline()
-        except:
+            model = open('/proc/device-tree/openprom/model', 'r').readline()
+        except IOError:
```

Comment:
specs of `open()` function at https://docs.python.org/3/library/functions.html#open says,
> Open file and return a corresponding file object. If the file cannot be opened, an OSError is raised.

Reply:
Thanks.Agreed. I made the changes accordingly please review.